testing/src/main/java/org/oppia/android/testing/AssertionHelpers.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.testing
+
+import org.junit.Assert.fail
+import kotlin.reflect.KClass
+import kotlin.reflect.full.cast
+
+/**
+ * Assertion helpers which are generic methods to be used at multiple places
```

Comment:
```suggestion
 * This file contains generic assertion helpers which are meant to be usable in any test.
```

This is a bit clearer. Also, this isn't a KDoc for assertThrows, it's more like a file-level comment. I suggest making this a non-KDoc (e.g. ``/* */`` rather than ``/** */``), and moving it above assertThrows (there should be a newline separating them).

Please also add a dedicated KDoc for assertThrows: this should explain what parameters should be passed to it, what it does, and what it returns.

Reply:
@BenHenning Thanks for the suggestions. I have updated it accordingly.